components/content/DownloadLogoPack.vue

```diff
@@ -0,0 +1,12 @@
+<template>
+    <a
+        download
+        href="/Kestra logo kit-20240422T105102Z-001.zip"
```

Comment:
```suggestion
        href="/kestra-logo-kit.zip"
```
please use a proper name